CONTRIBUTING.md

```diff
@@ -6,11 +6,13 @@ We follow the standard GitHub [fork & pull](https://help.github.com/articles/usi
 
 You're always welcome to submit your PR straight away and start the discussion (without reading the rest of this wonderful doc or the README.md). The goal of these notes is to make your experience contributing to Akka as smooth and pleasant as possible. We're happy to guide you through the process once you've submitted your PR.
```

Comment:
> contributing to Akka as smooth
is this just a oversight?

Reply:
@naosense : thanks, yes - missed that. In fact have missed a few more instances in this file, have fixed all of them. Please review.